spec/spec_helper.rb

```diff
@@ -71,3 +71,13 @@ def validate(arg)
   #     end
   config.infer_spec_type_from_file_location!
 end
+
+if Rails::VERSION::MAJOR > 3
```

Comment:
@iffyuva Do we need this conditional since we don't support Rails versions prior to 3?

Reply:
@aksh93 im planning to remove this in next major release.

Reply:
Alright.